app/src/main/java/org/p2p/wallet/newsend/NewSendFragment.kt

```diff
@@ -88,12 +84,29 @@ class NewSendFragment :
         binding.widgetSendDetails.setMaxButtonVisibility(isVisible)
     }
 
+    override fun setBottomButtonText(text: String) {
+        binding.buttonBottom.text = text
+    }
+
+    override fun setBottomButtonVisibility(isVisible: Boolean) {
```

Comment:
i suggest using setBottomButtonIsVisible naming due to 3 states of Visibility in Android